src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -231,6 +233,15 @@ public StreamingResponseBody streamEvents(
         };
     }
 
+    /**
+     * Every consumer identifies itself using a client-id to use its quota. The client id is a combination of
+     * application name and event type so that every application can consume up to the quota limit per partition.
+     * Leader partitions from a single event type are guaranteed to be located on different brokers.
```

Comment:
I made some changes in 0c78145 could you please take a look to see if better covers the behaviour?